Assignment/solution_student/css/style.css

```diff
@@ -1,27 +1,239 @@
 /*
-Project Name: HTML Template
-Client: Your Client
-Author: Your Name
-Developer: @poopsplat in Chicago
+Project Name: Landing Page
+Client: Relaxr
+Author: Haley 
+Developer: @needleinthehaley in Chicago
 */
 
 /******************************************
-/* BASE STYLES
+/* UTILITY
 /*******************************************/
-body {
-  color: black;
-  font-size: 16px;
-  background: white;
+
+*, *:before, *:after {
+  box-sizing: inherit;
+}
+html {
+  box-sizing: border-box;
+}
+.clearfix {
+  *zoom: 1;
+}
+.clearfix:before,
+.clearfix:after {
+    content: " ";
+    display: table;
+}
+.clearfix:after {
+  clear: both;
 }
 
+/*i started off with ALL intentions of utilizing the categories 
+outlined in the CSS template. that is SO hard to do when you are
+constantly trying to figure everything out. i didn't do it this 
+time, but if i have time i'll go back in and sort. i'm sure 
+duplication of code is annoying to review.*/
 /******************************************
-/* LAYOUT
+/* BASE STYLES
 /*******************************************/
+body {
+	color: black;
+	margin: 0;
+	font-size: 16px;
+	font-family: 'open sans', sans-serif;
+	width: 100%;
+}
 
-/******************************************
-/* PAGES
-/*******************************************/
+/****************************************/
+/*it looks like there is an opacity piece on top of my image, 
+but i can't figure it out! why is it so hazy?*/
+header {
+	background-image: url("../images/header_bg.jpg");
+	background-repeat: no-repeat;
+	height: 400px;
+	width: 100%;
+	background-size: cover;
+}
+
+footer {
+	background: #121212;
+	background-size: cover;
+	width: 100%;
+	height: 250px;
+	padding-top: 10px;
+	text-align: center;
+	opacity: 0.9;
+
+}
+
+/*the google font weights look very thick (compared to the PNG)
+i tweaked, but i read the doc! trust you, me. i try to follow
+directions*/
+h1, h2 { 
+	font-size: 24px;
+	color: #f0efef;
+}
+
+h1 {
+	font-weight: 700;	
+	margin-left: 50px;
+	margin-top: 0px;
+	margin-bottom: 0px;
+	padding-top: 20px;
+	padding-left: 50px;
+}
+
+h2 {
+	font-weight: 500;
+	margin-left: 300px;
```

Comment:
The big old margins that you're setting here and on the h1 were used with the intention of trying to get those elements centered within the header, which looks fine if your browser window is _just_ the right size, but anything smaller or larger then that and it's not centered ([example](http://d.pr/i/1eQl0)), **so**, instead of using margin, get rid of the margin, and just use `text-align: center` — and bam! Centered text! It's probably your best bet to apply `text-align: center;` to the `header` itself so that it's all centered, including the `a#now`, and then just go into your `h1` declaration to say `text-align: left;` since you want that logo to be off to the left still. Make sense?